backend/services/organisation_service.py

```diff
@@ -195,9 +195,7 @@ def get_organisation_stats(organisation_id: int, year: int) -> OrganizationStats
         ).filter(Project.organisation_id == organisation_id)
         if year:
             start_date = f"{year}/01/01"
-            projects = projects.filter(
-                Project.created.between(start_date, datetime.today())
-            )
+            projects = projects.filter(Project.created.between(start_date, func.now()))
```

Comment:
If this is not related to dependency updates and needs change. Let's do this in another PR.

Reply:
This was actually needed to fix a test. I didn't check to see if the test was failing prior to this PR, but I can check that.

Reply:
Without that change, `test_get_all_org_includes_stats_if_omit_stats_set_false` in tests/backend/integration/api/organisations/test_resources.py fails. I can drop it, if that is still desired.

Reply:
In that case, let's keep this change.

Reply:
I've reworded the commit in question.

Reply:
Thanks @tsmock.